examples/deck/example.js

```diff
@@ -1,21 +1,24 @@
 // @flow
+
 import window from 'global/window';
 import React, { Component } from 'react';
 import DeckGL, { MapController } from 'deck.gl';
 import { StaticMap } from 'react-map-gl';
 
-import { EditablePolygonsLayer } from 'nebula.gl';
+import { EditableGeoJsonLayer } from 'nebula.gl';
 
-import testPolygons from '../data/sf-polygons';
+import sampleGeoJson from '../data/sample-geojson.json';
 
 const initialViewport = {
   bearing: 0,
   height: 0,
-  latitude: 37.75,
-  longitude: -122.4,
+  // latitude: 70.13,
```

Comment:
Let's try to have some examples in places that are not SF.

Reply:
Agreed, and also more examples that show off more of its capabilities (e.g. editing a single feature as opposed to FeatureCollection). The layer-browser in deck.gl is stellar.